security-ldap-quickstart/src/main/resources/quarkus-io.ldif

```diff
@@ -0,0 +1,81 @@
+version: 1
```

Comment:
Was this file added by accident?

Reply:
Yes, I was plaing with and forgot that I added it.

Now removed